docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Filecoin FAQ
+description: Filecoin frequently asked questions.
+---
+
+# Filecoin FAQ
+
+[[toc]]
+
+## Getting Started
+
+### How do ordinary people, not geeks, get involved in this work?
+
+You can start from checking out the tutorials from [Protoschool](https://proto.school/#/tutorials)
+Participate in the hackathon and future events of [Filecoin Ignite](https://ignite.fil.events/)
+Check out the different tools to [build dApps on Filecoin](https://blog.textile.io/)
+
+### What are some of the primary use cases for Filecoin at mainnet launch?
+
+The primary use case for Filecoin at first will be more 'long term' storage applications. Archival, personal backups, etc. As well as applications like existing dApps that don't want to (or 'can't') store their data in traditional ways. There is fairly significant demand among current decentralized applications for a solution to how their data should be stored, and a good number of them already use IPFS (but resort to pinning data themselves, on their servers to ensure the data sticks around). These apps already have wallets built in usually, and adding some sort of micropayment for the data they use isn't too big of a stretch.
+
+### How can a website app be free on a network where retrieval is monetary?
+
+There are a good number of websites out there right now where file downloads are supported entirely through huge amounts of ads (the sort of thing where you click through three pages of ads before you get to your really slow download). All of those types of websites could be easily replaced with a Filecoin incentivized retrieval setup, paying small amounts of Filecoin directly for whatever files you were hoping to download (the biggest one here in my mind is when you're trying to download mods for games). There are also a bunch of large datasets out there that are hosted through amazons 'pay per download' s3 buckets, Filecoin retrieval easily competes with that too.
```

Comment:
@jasonleyser When we're writing FAQs, please remember to change the voice to not include "I" or "my" -- e.g. "in my mind". This is intended to be published as the official Filecoin project, not a particular individual.

Please replace with:
Several existing websites support file downloads entirely through ads. All of those types of websites could be easily replaced with a Filecoin incentivized retrieval setup, paying small amounts of FIL (Filecoin tokens) directly for whatever files you were hoping to download (e.g. trying to download mods for games). Several large datasets are hosted through Amazon's "pay per download" S3 buckets, which Filecoin retrieval could also easily augment or replace.